foundation.dd

```diff
@@ -0,0 +1,64 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(H4 The D Language Foundation is an organization devoted to advancing
+     open source technology related to the D programming language.
+)
+$(H3 History)
+
+$(P The D programming language was started by Walter Bright, as a 'better C++'
+and has grown slowly, but steadily, for more than 15 years. Recent improvements
+to the D development process have accelerated development of the language and
+shown the need for a governing organization that overlooks and arranges the
+future development of the D programming language.)
+
+$(H3 Current projects)
+
+The D foundation
+
+$(UL
+
+    $(LI is providing the various resources for the D community
+        (e.g. this website, dmd downloads, the D tour and more))
+    $(LI is an accepted
+        $(HTTPS summerofcode.withgoogle.com/organizations/5078256051027968/,
+        Google Summer of Code organization) and hosts four projects during
+        the summer of 2016.
+    )
+    $(LI is organizing the yearly $(HTTP dconf.org, D conference))
+)
+
+$(H3 Committee)
+
+The officers of the D Language Foundation are:
+
+$(UL
+    $(LI $(HTTP www.walterbright.com, Walter Bright), President)
+    $(LI $(HTTPS erdani.com/index.php/contact, Tudor Andrei Cristian Alexandrescu), Vice President and Treasurer)
+    $(LI $(HTTP http://acehreli.org, Ali Çehreli), Secretary)
```

Comment:
Yah, that was the one. I think this works in most browsers but `&Ccedil; ` would put us on the safe side. I'll pull anyway.